02_bubble/ailyntang/sort.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintln(out, bubble([]int{3, 2, 1, 5}))
+}
+
+func bubble(s []int) []int {
```

Comment:
Due to: https://github.com/anz-bank/go-course/blob/master/02_bubble/README.md
You should be working on slice copy (`Write a function that returns a sorted copy of int slice s using bubble sort`)

Please consider adding:
`ts := append(s[:0:0], s...)`
and working on `ts` instead of `s`

Reply:
Thanks for the pickup, I forgot about the copy.